scene/gui/scroll_bar.cpp

```diff
@@ -46,9 +46,6 @@ void ScrollBar::gui_input(const Ref<InputEvent> &p_event) {
 	ERR_FAIL_COND(p_event.is_null());
 
 	Ref<InputEventMouseMotion> m = p_event;
-	if (!m.is_valid() || drag.active) {
```

Comment:
MouseMotion does not call `scroll()` anywhere, which is what caused the regression I mentioned.

Reply:
Yeah, it calls `set_as_ratio` instead. I'll look into how to handle this.

Reply:
`scroll_to_ratio()` perhaps.

Reply:
Added checks around where `set_as_ratio` was called and send signal if it's scrolling

Reply:
`scroll_to_ratio()` is an idea, but I'm not sure it's called enough for that.